system/apps/blog/store/article-maintain/article-maintain.store.ts

```diff
@@ -0,0 +1,117 @@
+import { create } from 'zustand';
+import type {
+  ArticleMaintainFormData,
+  ArticleMaintainStore,
+  ArticleMaintainActions,
+} from './defs';
+import {
+  createArticle,
+  updateArticle,
+  getArticle,
+  getError,
+} from '@system/blog-api';
+import { form, required } from '@system/utils';
+
+const maintainForm = form<ArticleMaintainFormData>()({
+  title: [required],
+  description: [],
+  content: [],
+  lang: [],
+  thumbnail: [],
+});
+
+const useArticleMaintainStore = create<ArticleMaintainStore>(() => ({
+  is: 'idle',
+}));
+
+const change: ArticleMaintainActions['change'] = (key, value) => {
+  const get = useArticleMaintainStore.getState;
+  const set = useArticleMaintainStore.setState;
+
+  set({
+    form: maintainForm.set(get().form)({ [key]: value }),
+  });
+};
+
+const confirm: ArticleMaintainActions['confirm'] = async () => {
+  const set = useArticleMaintainStore.setState;
+  const get = useArticleMaintainStore.getState;
+
+  const form = maintainForm.confirm(get().form);
+
+  if (form.invalid) {
+    set({ form });
+    return;
+  }
+
+  set({ form, is: 'pending' });
+
+  try {
+    const get = useArticleMaintainStore.getState;
+    const url = get().url;
+    const isInCreation = !url;
+
+    if (isInCreation) {
+      await createArticle({
+        ...form.values,
```

Comment:
U nas w projekcie nie używa się trzech kropek :/. Jest to bardzo nieczytelne i jest to zła praktyka. 
Proszę usuń to z kodu :/